apps/web/frontend/components/projectMemberList/projectMemberList.tsx

```diff
@@ -60,7 +79,12 @@ export const ProjectMemberList = (props: ProjectMemberListProps) => {
                 {user.name}
               </TableCell>
               <TableCell>
-                <RoleLabel role={user.role} />
+                <RoleLabel
```

Comment:
make one for the label and another for the button to downgrade or upgrade. (Both buttons only usable for project admins)

Reply:
do not allow to upgrade or downgrade yourself